src/toolboxes/spline/Canvas.tsx

```diff
@@ -38,47 +47,47 @@ export class SplineCanvas extends Component<Props> {
     super(props);
     this.selectedPointIndex = -1;
     this.isDragging = false;
+    this.mode = Mode.draw;
   }
 
   drawSplineVector = (splineVector: XYPoint[], isActive = false): void => {
-    if (splineVector.length < 2) return;
+    if (splineVector.length === 0) return;
 
     const { canvasContext: context } = this.baseCanvas;
     const lineWidth = isActive ? 2 : 1;
-
-    // Clear the canvas
     context.lineWidth = lineWidth;
     context.strokeStyle = "#ff0000";
```

Comment:
Can we use the material UI colour names here? E.g. use the default primary and secondary colours?